### ISSUE TITLE
3.8.2 breaks com_modules when using access (administration and display)

### ISSUE THREAD

**Issue by aschkenasy** [joomla/joomla-cms#18839]
### Steps to reproduce the issue
In administration assign a module the "guest" access

### Expected result
module will be displayed to guests only.
mosule should be editable by admin from backend...
(just to state the obvious... such is the behavior in =< 3.8.1)

### Actual result
module disappears from administrator list (that's because the currently logged in admin is not a "guest")

**Comment by aschkenasy** [comment]
administrator/components/com_modules/models/modules.php

lines 315 - 320
are to blame
```
// Filter by current user access level.
// Get the current user for authorisation checks
$user   = JFactory::getUser();
$groups = implode(',', $user->getAuthorisedViewLevels());
$query->where('a.access IN (' . $groups . ')');
```

**Comment by ghost** [comment]
closed as duplicate Report #18542
